servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingStreamingHttpRequest.java

```diff
@@ -175,7 +175,14 @@ default InputStream payloadBodyInputStream() {
 
     /**
      * Returns a {@link BlockingStreamingHttpRequest} with its underlying payload transformed to {@link Buffer}s,
-     * with access to the trailers.
+     * with access to the <a href="https://tools.ietf.org/html/rfc7230#section-4.1.2">trailer</a>s.
+     * <p>
+     * Trailers are supported only when the {@link HttpProtocolVersion#HTTP_1_1 HTTP/1.1} version is used or above.
```

Comment:
`Trailers are supported only when the {@link HttpProtocolVersion#HTTP_1_1 HTTP/1.1} version is used or above.` -> `The minimum required version for trailers is {@link HttpProtocolVersion#HTTP_1_1 HTTP/1.1}`

Reply:
I've removed these comments because the behavior was reverted.